### PR TITLE
Update advisory.html

### PR DESCRIPTION
**Pull request by Jaredabera**
Added some goals of the carpentries yet to be decided. Do you think its important to add it here?

### CODE REVIEW

File: pages/advisory.html [hunk 1]
```diff
@@ -13,6 +13,10 @@ <h4>Current 2019-2020 Goals:</h4>
 <li>Inform the Library Carpentry community about the Advisory Group and that it is a resource for the community.</li>
 <li>Check-in with at least 5-10 Library Carpentry community members, see what they are up to following the workshops they hosted, helped with, and/or instructed.</li>
 <li>Draft letter of interest to explore pool of funding to empower libraries to start local Library Carpentry activities.</li>
+<li> Map Library Carpentry/Carpentries material to resume/CV/job descriptions/competencies.</li>
+<li>Discuss/pilot Library Carpentry/Carpentries ideas in library & information science programs.</li>
+<li>Describe other ways of teaching the material that don't require hosting an entire workshop.</li>
+<li>Survey to Library Carpentry workshop attendees, 6 months after, assess impact, gather stories.</li>
```
Comment (katrinleinweber) [inline pages/advisory.html:19]: With the ["upgrade" of LC to a Carpentries Lesson Program](https://carpentries.org/blog/2018/11/welcoming-library-carpentry/), don't [the overall assessments](https://carpentries.github.io/assessment/) cover LC as well?

@kariljordan: It's [referred to in some codes](https://github.com/carpentries/assessment/search?q=%22library+carpentry%22&unscoped_q=%22library+carpentry%22). Is a dedicated LC long-term report feasible?

Comment (libcce) [reply pages/advisory.html:19]: @katrinleinweber we are still in the process of voting, a few more votes from the Advisory Group need to be counted to determine what our priorities will be. It's still too early to even comment on your comment.

Comment (kariljordan) [reply pages/advisory.html:19]: The LC survey was adapted from Data Carpentry's survey and set up as a pilot to make sure the questions were relevant. We will be releasing learner assessment reports in Q4 of this year. After that release we will determine if LC's survey questions still align with the DC and SwC questions. We'll then move the survey over to Typeform. The current [Long-Term Survey](https://github.com/carpentries/assessment/blob/master/learner-assessment/surveys/longterm_survey_current.pdf) includes Library Carpentry.

**Comment by maneesha** [comment]
@libcce  can we merge this?

**Comment by libcce** [comment]
All Advisory Group members have voted. I've asked our chair @arieldeardorff to review the goals to see if the language needs some last minute changes. I'll post them here soon so @Jaredabera can update his PR and we can merge.

**Comment by libcce** [comment]
Thanks @Jaredabera for moving this forward. The goals are now posted https://github.com/LibraryCarpentry/librarycarpentry.github.io/commit/e117351bad78f90f93e55649fec80296f1aaad71 I'm closing this PR.

**Pull request closed without merging**